micropython/micropython-1.15-patchset/os.patch

```diff
@@ -1,6 +1,6 @@
 diff -Naur micropython-1.15/ports/unix/alloc.c micropython-phoenix/ports/unix/alloc.c
---- micropython-1.15/ports/unix/alloc.c	2021-04-14 07:35:31.000000000 +0200
-+++ micropython-phoenix/ports/unix/alloc.c	2021-04-14 13:27:45.000000000 +0200
+--- micropython-1.15/ports/unix/alloc.c	2021-04-18 16:27:07.000000000 +0200
++++ micropython-phoenix/ports/unix/alloc.c	2021-05-21 10:42:06.565325678 +0200
 @@ -49,21 +49,21 @@
      struct _mmap_region_t *next;
  } mmap_region_t;
```

Comment:
commented-out code is still here :P

Reply:
Patches cleared, comments not present now